cmd/helm/install.go

```diff
@@ -175,6 +175,7 @@ func addInstallFlags(cmd *cobra.Command, f *pflag.FlagSet, client *action.Instal
 	f.BoolVar(&client.Atomic, "atomic", false, "if set, the installation process deletes the installation on failure. The --wait flag will be set automatically if --atomic is used")
 	f.BoolVar(&client.SkipCRDs, "skip-crds", false, "if set, no CRDs will be installed. By default, CRDs are installed if not already present")
 	f.BoolVar(&client.SubNotes, "render-subchart-notes", false, "if set, render subchart notes along with the parent")
+	f.BoolVar(&client.PlainHTTP, "plain-http", false, "install charts from http registry")
```

Comment:
I think the `PlainHTTP` flag should be added here https://github.com/helm/helm/blob/866c6af20c1eef2f2db560528072af9cd9ed05ff/cmd/helm/flags.go#L53

It will then be available to the downloading commands.